Parsimony.Tests/Internal/TestOption.cs

```diff
@@ -7,32 +7,38 @@ namespace Parsimony.Tests.Internal
 {
     public class TestOption
     {
-        private readonly OptionName.Short _valueShortName =
-            OptionName.Parse("v") as OptionName.Short ?? throw new InvalidOperationException();
+        private static readonly OptionName.Short _valueShortName =
+            OptionName.Parse("v") as OptionName.Short ??
+                throw new InvalidOperationException($"Invalid option name for {nameof(_valueShortName)}.");
 
-        private readonly OptionName.Long _valueLongName =
-            OptionName.Parse("val") as OptionName.Long ?? throw new InvalidOperationException();
+        private static readonly OptionName.Long _valueLongName =
+            OptionName.Parse("val") as OptionName.Long ??
```

Comment:
just handles the possible null returned. it will never be null because that's a valid name, but the compiler doesn't like the "possible null assignment"

Reply:
Does the null forgiving op work here too?